src/main.ts

```diff
@@ -52,8 +53,15 @@ async function start () {
     })
   }
 
+  // Initialize Hotjar
+  if (window['hotjarId']) {
+    console.info('Initializing Hotjar...') // eslint-disable-line no-console
+    Vue.use(Hotjar, { id: window['hotjarId'], isProduction: true })
```

Comment:
I get nervous when I see "isProduction: true" in Dev code, but I'm assuming that nothing in Prod will be touched here. Can you confirm?

Reply:
Confirmed!

This is a Hotjar option to enable tracking. This is what we do with HJ in other places we use it. I don't know yet if we need tracking. See https://www.npmjs.com/package/vue-hotjar.